.github/CODEOWNERS

```diff
@@ -30,6 +30,10 @@ fabric-website/ @jordandrako
 #### Packages
 packages/charting/ @Raghurk
 packages/dashboard/ @Raghurk
+packages/charting/ @veeray
+packages/dashboard/ @veeray
+packages/charting/ @chankev
+packages/dashboard/ @chankev
```

Comment:
you just have to put the name with spaces like below

Reply:
changed to correct format